### PR TITLE
[SPMD] Add one Megatron simple e2e test and remove relative import

### PR DESCRIPTION
**Pull request by fduwjj**
We need a E2E Megatron style test to ensure that any further change down the road does not break it.

### CODE REVIEW

File: spmd/tensor/ops/pointwise_ops.py [hunk 1]
```diff
@@ -120,10 +118,7 @@
 #     "xlogy",
 # ]
 
-pointwise_ops = [
-    "aten.relu.default",
-    "aten.gelu.default",
-]
+pointwise_ops = ["aten.relu.default", "aten.gelu.default"]
```
Comment (wanchaol) [inline spmd/tensor/ops/pointwise_ops.py:121]: are we using the same formatter command? `black --line-length 80 spmd test/spmd`? it seems I use this command on my end but it never reformatted like this?

File: test/spmd/tensor/test_megatron_example.py [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+        distribute_tensor(m.net2.bias, device_mesh, replicate)
+    )
+    m = _replicate_input_tensor(m, device_mesh, replicate)
+    m.net2 = _aggregate_local_tensor(m.net2)
+    m.net1.weight.register_hook(
+        functools.partial(_gradient_hook, m.net1.weight)
+    )
+
+
+class DistTensorMegatronTest(DistTensorTestBase):
```
Comment (wanchaol) [inline test/spmd/tensor/test_megatron_example.py:80]: wouldn't this not working yet? I think CI will fail as the local patches not submitted yet

File: test/spmd/tensor/test_megatron_example.py [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+        optim.step()
+
+        torch.manual_seed(3)
+        inp = torch.rand(*inp_size).cuda(self.rank)
+        output = model(inp)
+        output_tp = model_tp(inp)
+        self.assertEqual(output, output_tp)
+
+
+if __name__ == "__main__":
```
Comment (wanchaol) [inline test/spmd/tensor/test_megatron_example.py:114]: maybe try removing this `main`? but probably pytest would still scan this test case lol

File: test/spmd/tensor/test_megatron_example.py (hunk 2 above)
Comment (fduwjj) [reply test/spmd/tensor/test_megatron_example.py:80]: Yes, I can merge this after your fix got landed?

File: test/spmd/tensor/test_megatron_example.py (hunk 3 above)
Comment (fduwjj) [reply test/spmd/tensor/test_megatron_example.py:114]: Hmmm you know what, maybe I can comment it out. And you can open it when your fix is ready. Let me do that.

File: spmd/tensor/ops/pointwise_ops.py (hunk 1 above)
Comment (fduwjj) [reply spmd/tensor/ops/pointwise_ops.py:121]: Yes I use "black --line-length 80 spmd test/spmd" and it just reformat like this...

**Review by wanchaol — APPROVED** [review]
thanks!!

**Pull request merged**